PysarenkoProj/src/test/java/mainPack/Test_Cases_HW

```diff
@@ -0,0 +1,139 @@
+/*
+Here is my candidates of automation test cases for shared project (http://v3.test.itpmgroup.com)
+---
+[I]. Logging in as user with proper credentials
+1.In X browser, go to link http://v3.test.itpmgroup.com;
+2. In "Email" field type proper Login;
+3. In "Password" field type proper Password;
+4. Click on "Vhod" button;
+
+ER: Address should be changed to "http://v3.test.itpmgroup.com" from "http://v3.test.itpmgroup.com/login"
+    AND
+    Same login, that was used for step 2 should be displayed on Main Panel bar
+    AND
+    User should see "Glavnaya" Page Title
+
+---
+[II]. Adding and deleting new "Apparat"
+1.In X browser, go to link http://v3.test.itpmgroup.com;
+2. In "Email" field type proper Login;
+3. In "Password" field type proper Password;
+4. Click on "Vhod" button;
+5. Click on "Slocari" in menu to open dropdown;
+6. Click on "Aparat" option from dropdown;
+7. On "Aparat" page click on "+" button;
+8. Fill in "Apparat number" with any integer;
+9. Fill in "Apparat Comment" with any string;
+10. Click on "Sozdat'" button -> user is transferred to Apparat page;
+11. Find on "Apparat" page just created "apparat" (by number and comment) -> click on it;
+12. Update values for fields "Apparat number" and "Apparat comment" with proper values and click on "Sohranit'" button -> user is transferred to Apparat page;
+13. Find on "Apparat" page just updated "apparat" (by number and comment) -> click on it;
+14. Click on "Udalit'" button -> user is transferred to "Apparat page";
+
+ER: Created and updated "aparat" can't be found on "Apparat page".
```

Comment:
а если удаление не работает? то аппарат останется? (такое удаление нужно относить в посткондишен)

Reply:
rewrite test cases - it will check only editing. Deletion is moved to Post-conditions.

If I understood correctly, it seems that deletion operations is better to check by it's individual auto test case. But is it a good practice to check several functionality as one gradual scenario with different functionalities?